build/release/Jenkinsfile

```diff
@@ -27,7 +27,7 @@ pipeline {
             setupGPGEnvironment()
             withCredentials([string(credentialsId: 'gpg.password', variable: 'GPG_PWD')]) {              
               def dryRun = isReleaseBranch() ? '' : '-DdryRun=true'
-              def args = " -P ossrhDeploy -Divy.engine.list.url=${params.engineListUrl} -Dgpg.project-build.password='${env.GPG_PWD}'";
+              def args = "-PossrhDeploy -Divy.engine.list.url=${params.engineListUrl} -Dgpg.project-build.password='${env.GPG_PWD}'";
```

Comment:
is this needed?

Reply:
yes, works better as we feed this args below into wrapped -Darguments="" ... there the leading whitespace is an issue